bld/CLMBuildNamelist.pm

```diff
@@ -4708,7 +4723,7 @@ sub write_output_files {
   $log->verbose_message("Writing clm namelist to $outfile");
 
   # Drydep, fire-emission or MEGAN namelist for driver
-  @groups = qw(drydep_inparm megan_emis_nl fire_emis_nl carma_inparm);
+  @groups = qw(drydep_inparm megan_emis_nl fire_emis_nl carma_inparm dust_emis_inparm);
```

Comment:
I'm not very familiar with how `@groups` works, but I noticed that I couldn't find `dust_emis_inparm` (or `carma_inparm`) elsewhere in the CLMBuildNamelist.pm.  I'm assuming this is fine, but wanted to double check.

Reply:
This is correct for the perl code. This just lists what namelists you want to write out and you only want to list each one once for the perl code. The namelist is referenced in the new namelist_definition file for dust_emis.